MoodDetectorWebApp/Views/Shared/_Layout.cshtml

```diff
@@ -8,8 +8,6 @@
     @Scripts.Render("~/bundles/jquery")
     @Scripts.Render("~/bundles/popper")
     @Scripts.Render("~/bundles/bootstrap")
-    <script type="text/javascript" src="https://download.affectiva.com/js/3.2.1/affdex.js"></script>
```

Comment:
Šitą visai ištrynei, ar tiesiog nematau, kur perkėlei?

Reply:
Yra `MoodDetectorWebApp/Views/Detector/Start.cshtml`. Be šito neveiktų detektorius.

Reply:
Ten yra šitas: <link href="~/Assets/css/detector-start.css" rel="stylesheet" type="text/css">
O kur download.affectiva.com........?

Reply:
Yra failo apačioje.